templates/hoover-deps.nomad

```diff
@@ -405,6 +405,67 @@ job "hoover-deps" {
   }
   {% endif %}
 
+  group "thumbnail-service" {
+    count = ${config.thumbnail_count}
+
+    ${ continuous_reschedule() }
+    ${ group_disk() }
+
+    task "thumbnail" {
```

Comment:
let's try to keep all the names predictable: the group is `thumbnail-service`, the task name is `thumbnail`, the fabio path is `_thumbnail`, but the key to get the image is `thumbnail-service` once again. Pick one descriptive name and use it all over, for example `thumbnail-generator`